package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "publish",
+  "version": "0.1.0-dev",
```

Comment:
no, the -dev still needs to be supplied in the package.json manually. Otherwise infusion would get 2.0.0-dev-dev....